### ISSUE TITLE
Skype on Prem but already using teams?

### ISSUE THREAD

**Issue by mdickens98** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#205]
We have S4B 2015 on prem and have already started using Teams so we have data in it. How do we go forward from this point? We have been looking into setting up Hybrid for our S4B but all our teams users appear in the accounts list in S4B online. We can't lose the data thats already been put into Teams.

**Comment by kenwith** [comment]
@mdickens98 - I think this is a very common scenario you describe and I know there is a lot of documentation being worked on and also some already published. Let me track that down and get back to you.

**Comment by kenwith** [comment]
@mdickens98 - I got the story from the product team and they are writing documentation that will be coming next month. I would recommend to wait for that documentation unless it is an emergency and I see if we can get the product team to expedite. In a nutshell, recommendation from product team is:

- If they want to do this now, they need to move the users to Skype for Business Online (using Move-CsUser cmdlet) and then use the cmdlet Grant-TeamsUpgradePolicy with Mode=TeamsOnly.
- If they are willing to wait, the upcoming CU8 for 2015 will have an updated version of Move-CsUser that will do this in one step. Documentation also coming regarding this scenario.

Closing this out and as soon as we get the new content the product team is working on we will get it published here in this repo right away. Thank you for your feedback.

CC @bjwhalen @CarolynRowe
